cpp/velox/compute/VeloxParquetDatasource.cc

```diff
@@ -97,7 +100,11 @@ void VeloxParquetDatasource::Init(const std::unordered_map<std::string, std::str
   auto properities =
       ::parquet::WriterProperties::Builder().write_batch_size(blockSize)->compression(compressionCodec)->build();
 
-  parquetWriter_ = std::make_unique<velox::parquet::Writer>(std::move(sink), *(pool_), 2048, properities);
+  std::unordered_map<std::string, std::string> configData({{velox::core::QueryConfig::kDataBufferGrowRatio, "2"}});
```

Comment:
Minor comment: Why not change Velox default value to 2 directly? :)

https://github.com/oap-project/velox/pull/238/files#diff-99eb5278674a972f5ec9278fd4df027dfb069b40bd40203e2e9d55e9f2a63dc6R244

Reply:
If changed the value to 2, there are several unit tests will failed by DataBuffer capacity check issue. So we keep the default value to 1 in velox.